library-best-practices/JLBP-1.md

```diff
@@ -4,7 +4,8 @@
 Use the minimum number of dependencies that is reasonable.
 Adding a dependency for a large amount of functionality may be ok,
 but avoid pulling in dependencies just to save a few lines of code,
-because every dependency is a liability.
+because every dependency is a liability inherited by your own
+dependents.
```

Comment:
(sorry I didn't notice on the first read-through)

It seemed like you were nudging me away from using any second-person pronouns :-P

Anyway it feels like a switch of target in the middle of this sentence. How about:

"because every dependency of a library is a liability of both that library and that library's consumers."

Reply:
How about consumers --> dependents?

In general second person is fine if the reader/you is really doing something. E.g. "When you type ls -a in the console..." However, in tech writing "you" sometimes becomes a general pronoun that can mean the reader/the compiler/the VM/the class/etc. depending on context. E.g. "If you depend on Guava then watch out for this problem." The reader doesn't depend on Guava. The reader's project does. 

Third person is often more specific helps the reader see what's really happening.